src/microstate.js

```diff
@@ -18,6 +19,7 @@ export default class Microstate {
    * @param {*} value
    */
   static create(Type, value) {
+    value = value && isCallable(value.valueOf) ? value.valueOf() : value;
```

Comment:
Is it possible for anything that is not `null` or `undefined` to not have a `valueOf()`

Reply:
@cowboyd not likely.

Reply:
I guess my point is, is it merely sufficient to just check that `value != null`

Reply:
It's trippy that `(10).valueOf()` is 10 and `10.valueOf()` is an exception. 
<img width="453" alt="screen shot 2018-03-09 at 8 15 34 am" src="https://user-images.githubusercontent.com/74687/37209148-1d41bd14-2372-11e8-8335-b5c5b53a2cff.png">